plugin/someteam.example.com/v1/chartinflator/ChartInflator

```diff
@@ -115,25 +116,18 @@ if [ -z "$releaseNamespace" ]; then
   releaseNamespace=default
 fi
 
-function doHelm {
-  $helmBin --home $helmHome $@
-}
-
-# The init command is extremely chatty
-doHelm init --client-only >& /dev/null
-
-if [ ! -d "$chartHome/$chartName" ]; then
-  doHelm fetch $chartVersionArg \
-      $chartRepoArg \
-      --untar \
-      --untardir $chartHome \
-      $chartNameArg
-fi
-
-doHelm template \
-    --name $releaseName \
-    --namespace $releaseNamespace \
-    --values $valuesFile \
-    $chartHome/$chartName
+case $HELM_VERSION in
+  v2*)
+    source $(dirname ${BASH_SOURCE[0]})/ChartInflator.v2.functions
+  ;;
+  v3*)
+    source $(dirname ${BASH_SOURCE[0]})/ChartInflator.v3.functions
+  ;;
+  *)
+    echo "[!] Unsupported 'helm' version '${HELM_VERSION}'" && exit 1
```

Comment:
please do echo to stderr, e.g.
echo "..." 1>&2 
Stdout is used by plugin to collect yamls and you'll not see anything in the error messages, like most likely happened in [1]. In combination with absence of '-c' option when you request version, you've got a very confusing output :) 

[1]
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kustomize/2321/kustomize-presubmit-master/1246934995168137221